src/index.js

```diff
@@ -34,7 +34,7 @@ app.post('/code', async (req, res, next) => {
     return res.status(422).send({
       code: 'incorrect-format',
       message: e.message,
-      errors: Array.isArray(e) ? e : null
+      errors: Array.isArray(e) ? e.map(error => error.message) : null
```

Comment:
I'd not proceed like this. These error objects contain useful information other than the message, such as the path to the key/value that's wrong. Is there any reason you want to do this?

Reply:
Example of error:

```
[ { keyword: 'additionalProperties',
    dataPath: '.components',
    schemaPath: '#/additionalProperties',
    params: { additionalProperty: 'parameters' },
    message: 'should NOT have additional properties' } ]
```

If I just see `should NOT have additional properties` in a long-enough document I'll have a hard time trying to figure out where the error is.

Reply:
I thought it'll be awkward to show an error as a JSON object, but thats fine. I'll rollback the change

Reply:
Yep. It's not the best design, but the way to improve it would be to create a nice UI to show the error. But that's another feature 😄